bluefog/torch/mpi_ops.py

```diff
@@ -715,18 +731,28 @@ def hierarchical_neighbor_allreduce_nonblocking(tensor: torch.Tensor,
     Bluefog processes for a given name. The reduction will not start until all processes
     are ready to send and receive the tensor.
 
+    In short, this API functions as the following equation.
```

Comment:
Nit: this function implements the following equation.

Reply:
Oh, functions here as verb?

Reply:
Yes. 😂  Changed anyway.